tests/certification/pubsub/azure/eventhubs/README.md

```diff
@@ -0,0 +1,82 @@
+    - Test: Confirms that all expected messages were received
+    - Test: Confirms that subscriber does not receive messages from the non-subscribed topic
+  - Test single publisher / multiple subscribers with same consumerID
+    - Start an app with 1 publisher and 2 subscribers
+    - The publisher publishes to 1 topic with 2 partitions (EventHub scalable consumer pattern)
+    - The subscriber is subscribed to 1 topic
+    - Test: Sends 100+ unique messages
+    - App: Simulates periodic errors
+    - Component: Retries on error
+    - App: Observes successful messages
+    - Test: Confirms that all expected messages were received
+    - Test: Confirms that each subscriber receive messages from only one partition
+  - Test single publisher / multiple subscribers with different consumerIDs
+    - Start an app with 1 publisher and 2 subscribers
+    - The publisher publishes to 1 topic with 2 partitions (EventHub scalable consumer pattern)
+    - The subscriber is subscribed to 1 topic
+    - Test: Sends 100+ unique messages
+    - App: Simulates periodic errors
+    - Component: Retries on error
+    - App: Observes successful messages
+    - Test: Confirms that all expected messages were received
+    - Test: Confirms that each subscriber receives messages from both partitions
+ - Test multiple publishers / multiple subscribers with different consumerIDs
+    - Start an app with 2 publishers and 2 subscribers
+    - The publisher publishes to 1 topic with 2 partitions (EventHub scalable consumer pattern)
+    - The subscriber is subscribed to 1 topic
+    - Test: Sends 100+ unique messages from each publisher
+    - App: Simulates periodic errors
+    - Component: Retries on error
+    - App: Observes successful messages
+    - Test: Confirms that all expected messages were received
+    - Test: Confirms that each subscriber receives messages from both partitions
+ - Test entity management
+   - Start a publisher and subscriber with a topic that does not exist
+   - Test: Confirm creation of topic/eventHub in given eventHub namespace
+   - Test: Send 100+ unique messasges to the newly created eventHub topic
+   - App: Observe successful messages
+   - Test: Confirm that subscriber receives all messages
+ - Test IOT Event Hub : [TODO]
+   - Start an app with 1 publisher and 1 subscriber
+   - The publisher publishes to 1 IOT EventHub with 1 partition 
+   - The subscriber is subscribed to 1 topic
+   - Test: Sends 100+ unique messages
+   - App: Simulates periodic errors
+   - Component: Retries on error
+   - App: Observes successful messages
+   - Test: Confirms that all expected messages were received
+   - Test: Confirms that all expected system properties are set
+### Authentication Tests 
+  - Test connection string based authentication mechanism
+  - Test AAD Service Principal based authentication
+    - Utilize a service principal with appropriate roles granted
+  - Test MSI based authentication : [TODO]
```

Comment:
Ignore MSI -- you can remove this. I have decided that it isn't necessary -- given the libraries we use, if things work with service principals they will also work with MSI.